azalea-client/src/events.rs

```diff
@@ -82,7 +95,7 @@ pub enum Event {
     /// # }
     /// # }
     /// ```
-    Packet(Arc<ClientboundGamePacket>),
+    Packet(ClientboundPacket),
```

Comment:
This would require you to have to start writing `Event::Packet(ClientboundPacket::Game(packet)) => ...` in the match statements which is kinda too verbose imo. Maybe it should be split into `Event::GamePacket` and `Event::ConfigurationPacket` instead.

Reply:
I didn't know you could even do that, I've been doing match.